src/libraries/System.Reflection.Emit/src/System/Reflection/Emit/FieldBuilderImpl.cs

```diff
@@ -101,386 +101,5 @@ public override void SetValue(object? obj, object? val, BindingFlags invokeAttr,
 
         public override bool IsDefined(Type attributeType, bool inherit) => throw new NotSupportedException(SR.NotSupported_DynamicModule);
         #endregion
-
-        internal sealed class MarshallingInfo
```

Comment:
Moved to `CustomAttributeWrapper.cs` as it is needed for `ParameterBuilderImpl` too